continuationsPluginExample/src/main/scala/examples/CPSExample.scala

```diff
@@ -215,3 +215,38 @@ def programSuspendContinuationNoSuspendContinuationVal: Int =
                 x.size + y.size + q.size + p.size + z
 
   fooTest(List(1))(List("A", "B"))(List(1, 1, 1))(List(1, 1, 1, 1))
+
+// runs only with scalac
```

Comment:
ContinuationsPluginExample should use scalac for compilation now. What happens when you uncomment this section?

Reply:
You can uncomment this section.

Reply:
On main, there should be a `bypass-zinc` project that aggregates the projects that need raw scalac compilation, and changes to the ci alias that keep it from crashing zinc in ci: https://github.com/47deg/TBD/blob/56cf64c3027716558597f2c283572a6f4f9f2d5f/build.sbt#L17-L42

Reply:
done in [77b9be8](https://github.com/47deg/TBD/pull/81/commits/77b9be8fdbbd5dfd4ba461589c6b9cae4acc0367)